readline/PKGBUILD

```diff
@@ -51,8 +35,11 @@ prepare() {
     msg "applying patch readline${_basever//./}-$(printf "%03d" $p)"
     patch -p0 -i $srcdir/readline${_basever//./}-$(printf "%03d" $p)
   done
-  patch -p1 -i $srcdir/readline-6.3-cygwin.patch
+  msg2 "$srcdir/readline-6.3-cygwin.patch"
```

Comment:
It probably will break some packages because of that. /usr/bin/msys-history7.dll /usr/bin/msys-readline7.dll

I'm not sure if there is a fix for that though.